install_config/http_proxies.adoc

```diff
@@ -115,6 +115,25 @@ However, this does not:
 NO_PROXY=*.example.com
 ----
 
+To deploy Hawkular Metrics on a proxied {product-title}
+environment, the following services should be included in `NO_PROXY`
```

Comment:
s/the following services should be included/do not include the following services